### PR TITLE
Install git-standup 

### PR DESCRIPTION
**Pull request by cyyyu**
Install with homebrew in macOS.

### CODE REVIEW

**Comment by cyyyu** [comment]
This pr comes because I didn't have git-standup before.
Nvm, I just saw the readme docs that you had mentioned this (installing with npm).  You can close or merge, up to you.
btw I like this project really, very funny.

**Comment by mojoaxel** [comment]
Please let's get rid of the standup-helper script and not making it even more platform dependent!

File: standup-helper.sh [hunk 1]
```diff
@@ -5,6 +5,14 @@ set -e
 # and run git-standup there directly from node (cwd did nothing), so
 # here we are.
 
+if ! type "git-standup" > /dev/null; then
+  echo "Seems like you haven't installed git-standup yet."
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "Installing git-standup with homebrew..."
+    brew install git-standup
```
Comment (hackjutsu) [inline standup-helper.sh]: I don't think we can assume clients have homebrew installed. At least we should suggest homebrew installation if the script fails to find it.

Comment (cyyyu) [inline standup-helper.sh]: yea you are right

Comment (notwaldorf) [inline standup-helper.sh]: Yeah, I definitely don't have homebrew, and this would fail badly.

**Comment by notwaldorf** [comment]
I'm gonna close this PR since we're trying to get away from git-standup altogether, and this adds even more platform dependencies :(

**Pull request closed without merging**